javascript/frameworks/cap/lib/advanced_security/javascript/frameworks/cap/RemoteFlowSources.qll

```diff
@@ -2,65 +2,29 @@ import javascript
 import advanced_security.javascript.frameworks.cap.CDS
 
 /**
- * A parameter of a handler registered for a service on an event. e.g.
- * ```javascript
- * this.on("SomeEvent", "SomeEntity", (req) => { ... });
- * this.before("SomeEvent", "SomeEntity", (req, next) => { ... }); // only `req` is captured
- * SomeService.on("SomeEvent", "SomeEntity", (msg) => { ... });
- * SomeService.after("SomeEvent", "SomeEntity", (msg) => { ... });
+ * Either a service is known and is exposed
```

Comment:
We now have a (or arguably the most) general definition of `HandlerParameter`, so all of the cases above are still relevant. Can you keep everything that was already there and put this sentence (`Either a service ... references in source`) in a (`Note: ...`) right below the example?

Reply:
cleaned it up to preserve more of what was originally there example wise!